### PR TITLE
Change video to use embed semantic and wrap some content to container

### PR DESCRIPTION
**Pull request by rizkyekoputra**

### CODE REVIEW

**Comment by rizkyekoputra** [comment]
@imrenagi  cek gan

File: src/routes/Home/components/CandidateSection/CandidateSection.js [hunk 1]
```diff
@@ -15,22 +31,19 @@ class CandidateSection extends React.Component {
...
+            </Container>
           </div>
         </div>
       )} />
+
     )
   }
 }
-
-CandidateSection.propTypes = {
```
Comment (imrenagi) [inline src/routes/Home/components/CandidateSection/CandidateSection.js]: Why did you remove this part? This part shouldn't be removed since it will check props that might be used by this components.

File: src/routes/Home/components/CandidateSection/CandidateSection.js [hunk 2]
```diff
@@ -1,11 +1,27 @@
 import React from 'react'
 import './candidatesection.css'
 import { Route } from 'react-router-dom'
-import { Button } from 'semantic-ui-react'
+import { Button, Container } from 'semantic-ui-react'
 import CandidateList from '../CandidateList'
-import PropTypes from 'prop-types'
```
Comment (imrenagi) [inline src/routes/Home/components/CandidateSection/CandidateSection.js]: You shouldnt remove this code
